config/routes.rb

```diff
@@ -128,10 +128,8 @@
   get '/api/deploys', to: 'deploys#index'
 
   get '/auth/github/callback', to: 'sessions#github'
-  get '/auth/google/callback', to: 'sessions#google'
-  post '/auth/ldap/callback', to: 'sessions#ldap'
-  get '/auth/gitlab/callback', to: 'sessions#gitlab'
-  get '/auth/bitbucket/callback', to: 'sessions#bitbucket'
+  get '/auth/:type/callback', to: 'sessions#omniauth_callback'
+  post '/auth/:type/callback', to: 'sessions#omniauth_callback'
```

Comment:
`any` did not work and using `match` with a placeholder did not work either 🤷‍♂️